compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/dma_loop_subsumption.mlir

```diff
@@ -463,12 +516,6 @@ module attributes {hal.executable.target = #executable_target_amdaie_xclbin_fb}
         scf.for %arg5 = %c0 to %c64 step %c1 {
           %2 = amdaie.npu.dma_cpy_nd %0([] [] [], [0, 0, 0] [1, 8, 16] [128, 16, 1])
         }
-        scf.for %arg6 = %c0 to %c1023 step %c1 {
-          %3 = amdaie.npu.dma_cpy_nd %0([] [] [], [0, 0] [8, 16] [16, 1])
-        }
-        scf.for %arg7 = %c0 to %c1024 step %c1 {
```

Comment:
I think it's useful to keep this one (or create a new test with a large loop extent) as the subsumption should happen in this case.